### PR TITLE
Vet Clinic: Update and Delete

### PR DESCRIPTION
**Pull request by Shisui6**
## Summary 🧾
- Update animals table
- Perform transactions on animals table
- Write queries using aggregate functions on animals table

## Screenshots 📷
- Transaction 1:
  
image

- Transaction 2:
  
image

- Transaction 3:
  
image

- Transaction 4:
  
image

- How many animals are there:
  
image

- How many animals have never tried to escape:
  
image

- What is the average weight of animals: 
  
image

- Who escapes the most, neutered or not neutered animals:
  
image

- What is the minimum and maximum weight of each type of animal:
  
image

- What is the average number of escape attempts per animal type of those born between 1990 and 2000:
  
image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', '12-05-2017', 5, TRUE, 11.00);
+
+-- Second round of data
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', '08-02-2020', 0, FALSE, 11.00);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', '15-11-2021', 2, TRUE, 5.70);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', '02-04-1993', 3, FALSE, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', '12-06-2005', 1, TRUE, 45.00);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Boarmon', '07-06-2005', 7, TRUE, 20.40);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Blossom', '13-10-1998', 3, TRUE, 17.00);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Ditto', '14-05-2022', 4, TRUE, 22.00);
```
Comment (AshakaE) [inline data.sql]: - [ ] Kindly cross check the `weight_kg` column values, some of them are supposed to be negative

File: queries.sql [hunk 2]
```diff
@@ -8,3 +9,77 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
...
+SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+
+ROLLBACK TO SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
```
Comment (AshakaE) [inline queries.sql:69]: - If all weights are positive like you have in `data.sql`, this query will not update any row 💡

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @Shisui6  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Git flow ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Shisui6👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**